letsencrypt/cli.py

```diff
@@ -668,8 +668,32 @@ def parse_args(self):
         parsed_args = self.parser.parse_args(self.args)
         parsed_args.func = self.VERBS[self.verb]
 
+        parsed_args.domains = self._parse_domains(parsed_args.domains)
         return parsed_args
 
+    def _parse_domains(self, domains):
+        """Helper function for parse_args() that parses domains from a
+        (possibly) comma separated list and returns list of unique domains.
+
+        :param domains: List of domain flags
+        :type domains: `list` of `string`
+
+        :returns: List of unique domains
+        :rtype: `list` of `string`
+
+        """
+
+        uniqd = None
```

Comment:
I wanted to leave the stuff like it was, for some possible corner cases deeper in the code. If no domain was defined by -d or config file, it would leave the namespace.domains as None previously, so I decided to follow that route here too.